Fix bungled normalization of plural/singular

ttung opened the pull request:
### Description
#85 was done incorrectly.  It's a Collection of SequencingReads.  Not a Collection of a singular SequencingRead.

#fml

### Test plan
upgraded and downgraded successfully.  repeated once more for good measure.

phoenixAja commented on docs/backend/schema.ddl:
```diff
@@ -148,7 +148,7 @@ Table SequencingProtocol {
 // describes a single set of sequence data from the sequencer. (FASTQ)
 // these should be replaced by the `HostFilteredSequenceRead` as soon as it is available so we are not storing any host reads
-Table SequencingReadCollection {
+Table SequencingReadsCollection {
```
```suggestion
Table SequencingReadsCollections {
```

phoenixAja commented on src/py/aspen/database/models/sequences.py:
```diff
@@ -84,14 +90,14 @@ class SequencingProtocolType(enum.Enum):
     entity_id = Column(Integer, ForeignKey(Entity.id), primary_key=True)
     sample_id = Column(Integer, ForeignKey(Sample.id), nullable=False)
-    sample = relationship(Sample, backref=backref("sequencing_reads"))
+    sample = relationship(Sample, backref=backref("sequencing_reads_collection"))
```
should this be `sequencing_reads_collections`?

ttung replied:
Miraculously, this is correct.  For any given sample, there can be at most one `sequencing_reads_collection`.

phoenixAja replied:
ok interesting lol

Commit:
Fix bungled normalization of plural/singular

### Description
#85 was done incorrectly.  It's a Collection of SequencingReads.  Not a Collection of a singular SequencingRead.

#fml

### Test plan
upgraded and downgraded successfully.
